### ISSUE TITLE
Problem running scripts

### ISSUE THREAD

**Issue by boggleboggle** [idleberg/vscode-applescript#35]
Hello,

I am just learning AppleScript and trying to run the following simple script:
tell application "Finder" to get the index of Finder window "Macintosh HD"

It runs fine in ScriptEditor giving me the answer: 1

The first time I ran it in VSCode with this extension, it worked. But now I get the following error in the results window:
osascript: extension-output-idleberg.applescript-#1-AppleScript: No such file or directory

Whatever script I run, I get this error. The scripts compile fine, but I can never run them.

I am using all the default options except the option to always show the results.

I have tried reinstalling the extension, restarting VSCode and recreating the files with the script, but none of those worked.

As I say, I am completely new to AppleScript, so maybe there is something obvious I am doing wrong, but I can't figure it out.

**Comment by idleberg** [comment]
Did you save the script?

**Comment by boggleboggle** [comment]
> Did you save the script?

I saved it in a Onedrive folder but the file is on the local drive. It could be a Onedrive-related issue, but I tried again with a new script saved to the local drive, but had the same problem.

One thing: I did delete the script and recreate it. Could that have messed up the references some how? However, I have reinstalled the extension, created new scripts, restarted VS Code and the computer all since then, and still get the problem.
